docs/.vuepress/theme/components/Header.vue

```diff
@@ -22,53 +22,61 @@
         target="_blank"
         rel="noopener noreferrer"
         title="Fork This Page (GitHub)"
-        class="sm-hide"
+        class="sm-hide nav-link"
       >
-        <img alt="GitHub" class="hide-dark" src="../images/icon-github.svg" />
-        <img
-          alt="GitHub"
-          class="show-dark"
-          src="../images/icon-github-white.svg"
-        />
+        <icon name="github" />
       </a>
       <span
-        class="pointer view-mode"
+        class="pointer view-mode nav-link"
         tabindex="0"
         @keydown.enter="$emit('toggle-mode')"
         @click="$emit('toggle-mode')"
         :aria-label="'Toggle View Mode'"
       >
-        <img
-          alt="Switch to Dark Mode"
-          class="hide-dark"
-          src="../images/icon-sun.svg"
-        />
-        <img
-          alt="Switch to Light Mode"
-          class="show-dark"
-          src="../images/icon-moon.svg"
-        />
+        <icon name="lightmode" class="hide-dark" />
```

Comment:
Rather than having 2 DOM elements & additional classes (`hide-dark` & `show-dark`) to handle the display logic, I'd rather just pass `darkMode` property from the Layout component into Header, then update the icon `name` based on that to present the correct icon. Make sense? You can see an example in Layout where we pass that property to the Hero component.

Reply:
Ah I'd missed that, I wasn't entirely sure how to handle it & I prefer what you suggest. Ty!